Upstream-Status: Submitted [https://github.com/danthedeckie/simpleeval/pull/119]
Signed-off-by: Ross Burton <ross.burton@arm.com>

From 08bc99211aec32882d5f1c83896eb5d72b9fb125 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Mon, 23 May 2022 11:35:00 +0100
Subject: [PATCH] pyproject.toml: remove build as a build-dependency

pypa/build is one possible build tool, but not the only build tool.  Forcing build to be present as a build dependency is meaningless: either the user is using build so obviously have build, or are using something else so having build is irrelevant.
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 052a55a..416482b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=30.3.0", "wheel", "build"]
+requires = ["setuptools>=30.3.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.black]
